FFXCutsceneRemover/Components/CutsceneRemover.cs

```diff
@@ -154,6 +154,52 @@ public void MainLoop()
                         Game.Resume();
                     }
 
```

Comment:
This was tricky but seems to work, since Brotherhood can be anywhere in the Equipment menu, we have to loop over every item (22 bytes for each) and find it so we can apply the buffs and skip the cutscene.